landoui/template_helpers.py

```diff
@@ -250,27 +255,27 @@ def repo_path(repo_url):
 
 
 @template_helpers.app_template_filter()
-def graph_width(cols):
+def graph_width(cols: int) -> int:
     return GRAPH_DRAWING_COL_WIDTH * cols
 
 
 @template_helpers.app_template_global()
-def graph_height():
+def graph_height() -> int:
     return GRAPH_DRAWING_HEIGHT
 
 
 @template_helpers.app_template_filter()
-def graph_x_pos(col):
+def graph_x_pos(col: int) -> int | float:
```

Comment:
@zzzeid due to the division here the type checker pointed out it is possible for this value to be a `float`. That's probably a bug, right?

Reply:
Division always results in a float, at least as of Python 3 and IIRC.

```
>>> type(1/1)
<class 'float'>
```

A `float` plus an `int` is also always a `float`.

Reply:
That is to say,

```suggestion
def graph_x_pos(col: int) -> float:
```

Reply:
I understand why it's `float`. I think the way the code is set up this function should always be returning `int`. I think we need to change the division to `//`, or add a constant for `7` since that's what the value will always evaluate to anyways.